packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.js

```diff
@@ -128,9 +145,9 @@ describe('ChangeEventPlugin', () => {
     var input = ReactTestUtils.renderIntoDocument(
```

Comment:
This won't work because it's not actually rendering into document (yes, the naming of that method is pretty bad). That's why reviews in other similar tests added setting up a container and adding it to the body before every test.